### PR TITLE
Scaffold indexes

### PR DESCRIPTION
**Pull request by TimvdLippe**
Warning: this is a big PR. Please review one commit at a time.

Most notably:
* A new version of `polymer-build` to fix some minification and issues
* Rename files (large amount of diff, but should be few changes)
* Introduce a new script `scaffold-indexes.js`. This file takes each top custom element per path and generates a special index.html. This makes sure that loading is blazingly fast.
* Removed a lot of unnecessary usages of `paper-`elements by normal HTML elements.
* Rewrote program to use a table instead. This is semantically correct and fixes styling
* Remove `app-header` and friends and replace with a simply styled `<header>`
* Remove bindings in pages and replace with actual content. This is because we can not use the binding system when scaffolding. It is also faster :tada: 
* Add a yarn file as we/I use Yarn

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,51 +4,42 @@ This is the code for the Symposium website. This website provides information ab
...
 
 ```
-$ polymer build
+$ yarn run build
 ```
 
-This will create a `build/` folder with `bundled/` and `unbundled/` sub-folders
-containing a bundled (Vulcanized) and unbundled builds, both run through HTML,
-CSS, and JS optimizers.
+This will create a `build/` folder containing an unbundled builds,
```
Comment (martijnjanssen) [inline README.md:38]: Does it create one or two builds here? Isn't it supposed to be a singular?

**Review by martijnjanssen — CHANGES_REQUESTED** [review]
Can't we use lambdas in the js? I would assume we can since we are now using the babel compiler. Should we also have a [.babelrc](http://babeljs.io/docs/usage/babelrc/) in the project now?

Are we actually using the slider that is listed in the imports?

As for visually testing, it all looks pretty good, still some things bother me:
- ~The min-width of the content is too wide for all mobile devices~
- ~The program-table seems to be missing~

Other than that, it looks solid!

File: src/symposium-homepage.html [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <link rel="import" async href="../bower_components/google-map/google-map.html">
 <link rel="import" async href="../bower_components/google-map/google-map-marker.html">
 <link rel="import" async href="../bower_components/iron-icon/iron-icon.html">
+<link rel="import" async href="../bower_components/iron-icons/iron-icons.html">
```
Comment (martijnjanssen) [inline src/symposium-homepage.html]: Nice, I just noticed this one.

**Comment by TimvdLippe** [comment]
@martijnjanssen We are using babeli as minifier, we do not transpile. Therefore we can sadly not use ES6 features yet.

Could you clarify the min-width comment?

Yes the program table isn't working due to an error with resolution. Trying to look for a solution atm

**Comment by martijnjanssen** [comment]
The min width is now fixed, including some layout improvements for small devices. 

The program page is now fixed in #53. Still have to add the shadow to the program section though. Are we merging this PR first or are we merging that one into this one? I would prefer merging #53 in this one and then fixing the shadow on the timetable.

**Comment by martijnjanssen** [comment]
@TimvdLippe The min width is now fixed, including some layout improvements for small devices. 

The program page is now fixed in #53. Still have to add the shadow to the program section though. Are we merging this PR first or are we merging that one into this one? I would prefer merging #53 in this one and then fixing the shadow on the timetable.

**Review by martijnjanssen — APPROVED** [review]

**Pull request merged**